src/fu-util.c

```diff
@@ -620,8 +510,7 @@ fu_util_modify_remote (FuUtilPrivate *priv,
 		return FALSE;
 
 	/* show some kind of warning when enabling download-type remotes */
```

Comment:
Since the warning turns on for non download remotes too, adjust this comment.

Reply:
I moved the check for DOWNLOAD type remotes to the serverside.

Reply:
Ah OK, I see.